tests/unit/test-cmd/test.sign.js

```diff
@@ -141,6 +200,68 @@ describe('sign', () => {
     }
   ));
 
+  it(
+    "doesn't allow a custom ID when no ID in manifest.json with submission api",
+    () => withTempDir(
+      (tmpDir) => {
+        const customId = 'some-custom-id';
+        const stubs = getStubs();
+        return sign(
+          tmpDir, stubs,
+          {
+            extraArgs: {
+              id: customId,
+              useSubmissionApi: true,
+            },
+            extraOptions: {
+              preValidatedManifest: manifestWithoutApps,
+            },
+          })
+          .then(makeSureItFails())
+          .catch(onlyInstancesOf(UsageError, (error) => {
+            assert.match(error.message, /Cannot set custom ID some-custom-id/);
+            assert.match(
+              error.message,
+              /requires a custom ID be specified in the manifest/);
+          }));
+      }
+    ));
+
+  it.skip(
```

Comment:
Why is this test skipped?

Reply:
It's not working and I don't really understand why 😞 - it's doing a similar thing to the existing test that works with the ID so either I'm doing something wrong, or (it's my suspicion that) the existing testing for the ID file being read is broken.